### PR TITLE
project up: wait until web server is ready

### PR DESCRIPTION
**Pull request by Chive**
- wait until web server is ready when running ``aldryn project up``
- also start project when running ``aldryn project open`` if it's not running after user confirmation
- remove ``.env`` file generation. this is now handled by https://github.com/aldryn/project-template/pull/3

### CODE REVIEW

**Comment by stefanfoulis** [comment]
:+1:

File: aldryn_client/localdev/main.py [hunk 1]
```diff
@@ -288,9 +282,32 @@ def open_project(open_browser=True):
...
+        fg='green'
+    )
+
+    click.secho('Waiting for project to start..', fg='green', nl=False)
+    # wait 30s for runserver to startup
+    seconds = 30
+    for attempt in range(seconds):
+        click.secho('.', fg='green', nl=False)
+        try:
+            response = requests.head(addr)
```
Comment (czpython) [inline aldryn_client/localdev/main.py]: this assumes that addr is always handled by a view no? otherwise 404

Comment (Chive) [inline aldryn_client/localdev/main.py]: damn, i thought ``response.ok`` would ignore 400 error codes

Comment (Chive) [inline aldryn_client/localdev/main.py]: i'll just remove the response check. that's good enough.

File: aldryn_client/localdev/main.py [hunk 2]
```diff
@@ -288,15 +284,52 @@ def open_project(open_browser=True):
...
+    click.secho('Waiting for project to start..', fg='green', nl=False)
+    # wait 30s for runserver to startup
+    seconds = 30
+    for attempt in range(seconds):
+        click.secho('.', fg='green', nl=False)
+        try:
+            requests.head(addr)
+            break
+        except requests.ConnectionError:
+            sleep(1)
```
Comment (czpython) [inline aldryn_client/localdev/main.py]: I'd feel better if you add try:except:else where else has the break :dancer:

**Comment by czpython** [comment]
:+1:

**Pull request merged**